sleap/gui/app.py

```diff
@@ -1003,9 +1008,101 @@ def _add_button(to, label, action, key=None):
+        hb.addWidget(self.skeletonTemplates)
+        hb.addWidget(QLabel("skeleton template"))
+        hbw = QWidget()
+        hbw.setLayout(hb)
+        vb.addWidget(hbw)
+
+        hb = QHBoxLayout()
+        preview_image = QtGui.QPixmap(get_package_file("sleap/gui/no-preview.png"))
+        self.skeleton_preview_image = QLabel("Preview Skeleton")
+        self.skeleton_preview_image.setPixmap(preview_image)
+        hb.addWidget(self.skeleton_preview_image)
+        hb.setAlignment(self.skeleton_preview_image, Qt.AlignLeft)
+
+        self.skeleton_description = QLabel(
+            f'<strong>Description:</strong> {self.state["skeleton_description"]}'
+        )
+        self.skeleton_description.setWordWrap(True)
+        hb.addWidget(self.skeleton_description)
+        hb.setAlignment(self.skeleton_description, Qt.AlignLeft)
+
+        hbw = QWidget()
+        hbw.setLayout(hb)
+        vb.addWidget(hbw)
+
+        self.state.connect(
+            "skeleton_description",
+            lambda label: self.skeleton_description.setText(
+                f"<strong>Description:</strong> {label}"
+            ),
+        )
+
+        def updatePreviewImage(preview_image: bytes):
+            if preview_image is None:
+                preview_image = QtGui.QPixmap(
+                    get_package_file("sleap/gui/no-preview.png")
+                )
+            else:
+
+                preview_image = decode_preview_image(preview_image)
+
+                # Create a QImage from the Image
+                preview_image = QtGui.QImage(
+                    preview_image.tobytes(),
+                    preview_image.size[0],
+                    preview_image.size[1],
+                    QtGui.QImage.Format_RGBA8888,  # Format for RGBA images (see Image.mode)
+                )
+
+                preview_image = QtGui.QPixmap.fromImage(preview_image)
+
+            self.skeleton_preview_image.setPixmap(preview_image)
+
+        self.state.connect("skeleton_preview_image", updatePreviewImage)
+
+        gb.setLayout(vb)
+        skeleton_layout.addWidget(gb)
+
+        def update_skeleton_preview(idx: int):
+            if idx > 0:
+                skel = Skeleton.load_json(skeletons_json_files[idx - 1])
+            else:
+                skel = Skeleton()  # TODO: handle custom?
+            self.state["skeleton_description"] = (
+                f"{skel.description}<br><br>"
+                f"<strong>Nodes ({len(skel)}):</strong> {', '.join(skel.node_names)}"
+            )
+            updatePreviewImage(skel.preview_image)
+
+        self.skeletonTemplates.currentIndexChanged.connect(update_skeleton_preview)
+
+        hb = QHBoxLayout()
+        _add_button(hb, "Load Skeleton", self.commands.openSkeleton)
+        _add_button(hb, "Save Skeleton", self.commands.saveSkeleton)
+
+        hbw = QWidget()
+        hbw.setLayout(hb)
+        skeleton_layout.addWidget(hbw)
 
         gb = QGroupBox("Nodes")
```

Comment:
We no longer need this to be in a `GroupBox` since it is contained in its own tab (same for the "Edges" `GroupBox`)